pages/date-range-picker/embedded.page.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import { Box, DateRangePickerProps, Link, SpaceBetween, FormField } from '~components';
+import { DateRangePickerEmbedded } from '~components/date-range-picker/embedded';
+import { i18nStrings, relativeOptions } from './common';
+import { getBrowserTimezoneOffset } from '~components/date-range-picker/time-offset';
+
+export default function DatePickerScenario() {
+  const [value, setValue] = useState<DateRangePickerProps['value']>({
+    type: 'absolute',
+    startDate: '2018-01-09T12:34:56Z',
+    endDate: '2018-01-19T15:30:00',
+  });
+
+  const timeOffset = getBrowserTimezoneOffset();
```

Comment:
Why the timeOffset is defined explicitly for the embedded example?

Reply:
The embedded component does require a timeOffset. However we could instead set the offset within the component itself instead of here